### PR TITLE
changes made in ipa-apartment on robot

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

File: cob_default_env_config/ipa-apartment/navigation_goals.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 #[x,y,theta]
 home: [0, 0, 0]
+entrance: [-0.180, -3.599, 3.141]
+charger: [1.513, -0.375, 1.490]
+
+uhr: [-10.299, 12.889, 1.479]
```
Comment (floweisshardt) [inline cob_default_env_config/ipa-apartment/navigation_goals.yaml]: please delete

**Comment by NadiaHG** [comment]
CR: OK

File: cob_default_env_config/ipa-apartment/navigation_goals.yaml (hunk 1 above)
Comment (benmaidel) [inline cob_default_env_config/ipa-apartment/navigation_goals.yaml]: which one do you mean? only uhr?
I would agree on removing uhr, but for the others i think it is ok.

Comment (floweisshardt) [inline cob_default_env_config/ipa-apartment/navigation_goals.yaml]: yes, only uhr. entrance and charger sounds useful

**Comment by benmaidel** [comment]
removed uhr from navigation goals

**Comment by floweisshardt** [comment]
CR: OK

**Comment by benmaidel** [comment]
ready for merge!

**Pull request merged**